projects/mapserver/project.yaml

```diff
@@ -8,12 +8,4 @@ vendor_ccs:
   - "sdlime@gmail.com"
 auto_ccs:
   - "ajsinghyadav00@gmail.com"
-fuzzing_engines:
-  - libfuzzer
```

Comment:
You don't want afl and honggfuzz?

Reply:
reading https://google.github.io/oss-fuzz/getting-started/new-project-guide/#fuzzing_engines : " By default, libfuzzer, afl, honggfuzz, and centipede are used" , my understanding is that the absence of the section, means that the 4 before mentioned engines are used ?